indico/modules/events/abstracts/controllers/abstract_list.py

```diff
@@ -226,18 +227,23 @@ def _process(self):
         abstract_persons = (AbstractPersonLink.query
                             .filter(AbstractPersonLink.abstract.has(self._membership_filter))
                             .all())
+        registered_persons = get_registered_event_persons(self.event)
         abstract_persons_dict = defaultdict(lambda: {'speaker': False, 'submitter': False, 'primary_author': False,
                                                      'secondary_author': False})
         for abstract_person in abstract_persons:
-            dict_key = abstract_person.person.user if abstract_person.person.user else abstract_person.person
-            person_roles = abstract_persons_dict[dict_key]
-            person_roles['speaker'] |= abstract_person.is_speaker
-            person_roles['primary_author'] |= abstract_person.author_type == AuthorType.primary
-            person_roles['secondary_author'] |= abstract_person.author_type == AuthorType.secondary
+            user_or_person = abstract_person.person.user if abstract_person.person.user else abstract_person.person
+            person_dict = abstract_persons_dict[user_or_person.id]
+            person_dict['identifier'] = user_or_person.identifier
+            person_dict['full_name'] = user_or_person.full_name
+            person_dict['email'] = user_or_person.email
+            person_dict['affiliation'] = user_or_person.affiliation
+            person_dict['speaker'] |= abstract_person.is_speaker
+            person_dict['primary_author'] |= abstract_person.author_type == AuthorType.primary
+            person_dict['secondary_author'] |= abstract_person.author_type == AuthorType.secondary
+            person_dict['registered'] = abstract_person.person in registered_persons
         for submitter in submitters:
-            abstract_persons_dict[submitter]['submitter'] |= True
-        return jsonify_template('events/abstracts/management/abstract_person_list.html',
-                                event_persons=abstract_persons_dict, event=self.event)
+            abstract_persons_dict[submitter.id]['submitter'] |= True
```

Comment:
This is causing the blank entries from my screenshot - we're adding people to the list by ID, but previously we added the user itself and thus had the name etc. So when adding someone here who's not in the dict yet, it's now necessary to populate the name etc. as well.

Reply:
My bad, I didn't realize there could be submitters that are not authors. Everything should be fixed now